src/containers/LeftPanel.js

```diff
@@ -0,0 +1,96 @@
+import React, {Component} from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import Button from '../components/Button'
+import Input from '../components/Input'
+
+import { 
+  selectDocument,
+  changeToAddMode,
+} from '../reducers/documents'
+
+import AddIcon from '../assets/images/plus.png'
+
+import './css/LeftPanel.css'
+
+class LeftPanel extends Component {
+
+  constructor(props){
+    super(props)
+    this.state = { searchText: '' }
+  }
+
+  handleSearch = (event) => {
+    this.setState({ searchText: event.target.value })
+  }
+
+  render() {
+    return (
+      <div className="left_panel-container">
+        {this.renderTopContent()}
+        {this.renderList()}
+      </div>
+    );
+  }
+
+  renderList() {
+    const { documents, selected } = this.props
+    let filteredDocuments = this.state.searchText.length < 2 ? documents :
+      filteredDocuments = documents.filter((doc) => {
+        return doc.title.includes(this.state.searchText)
+      })
+    return(
+      <div className="left_panel-list">
+        {
+         filteredDocuments.map(doc => {
+            return(
+              <div 
+                key={doc.id} 
+                className={`left_panel-list__document-title ${doc.id === selected.id ? "left_panel-list__document-title--selected" : ""}`} 
```

Comment:
Better define selectedClass variable above the return statement, if inline like this it's not so readable